xml/troubleshooting.xml

```diff
@@ -729,59 +726,12 @@
         </listitem>
       </itemizedlist>
       <para>
-        In all cases that do not involve external network problems, the
-        solution is to reboot the system into single-user mode and repair the
-        configuration before booting again into operating mode and attempting
-        to log in again. To boot into single-user mode:
+        In all cases that do not involve external network problems, the solution
+        is to reboot the system into the rescue mode and repair the
+        configuration before booting again into operating mode and attempting to
+        log in again. To boot into the rescue targe, follow the procedure
+        outlined in <xref linkend="proc-trouble-login-no"/>.
```

Comment:
IHMO this does not make sense. If we first say root can still log in (lines 681 + 682), there should be no reason to reboot. If root can still log in, you could simply run `systemctl emergency`  or `systemctl rescue`.

How about something along the lines of:

````suggestion
        is to log in as &rootuser; and repair the configuration. If you cannot log
        in to the running system, reboot it into the rescue mode as outlined in
        <xref linkend="proc-trouble-login-no"/>.